docs/inso-cli/cli-command-reference/inso-export-spec.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: article-detail
+title:  inso export spec
+category: "CLI Command Reference"
+category-url: inso-cli
+---
+
+This command will extract and export the raw OpenAPI specification from the data store. If the `--output` option is not specified, the specification will print to console.
+
+### Command
+
+`inso export spec [identifier]`
+
+Export the given specification, the user will be prompted to select a specification if one is not passed as an option. This can also be a file on your filesystem.
+
+`identifier` can be a specification name, or id.
+
+### Options
+
+{:.table .table-striped}
+Option  | Alias | Description
+----- | ------ | -------
+`--output <path>` | -o | Save the specification to a file in the working directory.
+
+### Examples
+
+When running in the example git-repo directory
```

Comment:
The [examples in the README](https://github.com/Kong/insomnia/tree/develop/packages/insomnia-inso#run-test-examples) link to this example git-repo which exists [here](https://github.com/Kong/insomnia/tree/develop/packages/insomnia-inso/src/db/fixtures/git-repo). We should do that here too.

This line exists in each command document so it'd just be a copy-paste.